### PR TITLE
Style Guide update July (additions to terminology)

### PR DESCRIPTION
**Pull request by dariavladykina**
Updated definitions of drop-down list, list box #206 
Updated definitions of drop-down list, list box #153  
Added spelling of multi-version (adj.) #122
Added definitions of screen, view #121 
Added spelling of bare metal, bare-metal (adj.) #80

### CODE REVIEW

**Review by jfaltenbacher — APPROVED** [review]
LGTM

**Review by jfaltenbacher — APPROVED** [review]
LGTM

File: xml/docu_styleguide.terminology.xml [hunk 1]
```diff
@@ -1795,8 +1821,9 @@
      </row>
      <row>
       <entry>usage</entry>
-      <entry>utilisation [British], utilization</entry>
-      <entry>noun</entry>
+      <entry/>
+      <entry>noun; the way in which something is used, or the amount of
+        it that is used; see also <emphasis>utilization</emphasis></entry>
```
Comment (cwickert) [inline xml/docu_styleguide.terminology.xml:1826]: Indention
```suggestion
       it that is used; see also <emphasis>utilization</emphasis></entry>
```

Comment (cwickert) [reply xml/docu_styleguide.terminology.xml:1826]: Please also fix the other indention issues.

File: xml/docu_styleguide.terminology.xml [hunk 2]
```diff
@@ -1820,11 +1847,27 @@
...
       <entry>video DVD</entry>
       <entry>Video DVD, Video-DVD, DVD video</entry>
       <entry>noun</entry>
      </row>
+     <row>
+       <entry>view</entry>
+       <entry/>
+       <entry>noun; a reusable set of user interface widgets (text fields, 
+             date time pickers, buttons) that serve as an interface for 
+             user interaction with an object or service</entry>
```
Comment (cwickert) [inline xml/docu_styleguide.terminology.xml:1869]: How about simplifying this:
```suggestion
      <entry>noun; a reusable set of user interface widgets that serve as an
       interface for user interaction</entry>
```

File: xml/docu_styleguide.terminology.xml [hunk 3]
```diff
@@ -1820,11 +1847,27 @@
               <emphasis>kernel space</emphasis>
       </entry>
      </row>
+     <row>
+        <entry>utilization</entry>
+        <entry>utilisation [British]</entry>
+        <entry>noun; an act or instance of making practical or 
+                profitable use of something, especially in 
+                <emphasis>CPU utilization</emphasis>, 
+                <emphasis>memory utilization</emphasis>
```
Comment (cwickert) [inline xml/docu_styleguide.terminology.xml:1856]: This is the definition from dictionary.com. While it is certainly right, I don't think it goes together well with CPU or memory utilization. Unfortunately I don't have a better definition atm.

File: xml/docu_styleguide.terminology.xml [hunk 4]
```diff
@@ -886,7 +897,9 @@
       <entry>list box</entry>
       <entry>list, list field</entry>
       <entry>noun; GUI element that is a list showing multiple elements even
-              before interacting with it
+              before interacting with it. To help the user interact with UI,
+              refer to a list box by its label and the word
+              <emphasis>list</emphasis>, not <emphasis>list box</emphasis>.
```
Comment (cwickert) [inline xml/docu_styleguide.terminology.xml:902]: I don't think we need the last sentence as it has little to do with terminology. If we refer to drop-downs by label, we should do so consistently for other widgets as well. Whether or not to use labels is a different question (and one we should certainly discuss in a separate issue).

**Review by cwickert — COMMENTED** [review]
First, please fix the indention issues. It's kind of funny if the style guide includes how to format XML, but does not follow it's own rules. ;-) I'd like us to think of the Style Guide as gold standard and prime example for our documentation.

Then, have a look at the other comments. Some are suggestions that can easily be applied, others don't have to. I think we can simplify many of the terms. This is not a dictionary, so I don't think we need a complete definition for everything, just what is important in the context of our docs and helps writers to tell one thing from another.

**Review by jfaltenbacher — APPROVED** [review]

File: xml/docu_styleguide.terminology.xml (hunk 4 above)
Comment (dariavladykina) [inline xml/docu_styleguide.terminology.xml:902]:
```suggestion
              before interacting with it
```

File: xml/docu_styleguide.terminology.xml [hunk 5]
```diff
@@ -149,6 +149,17 @@
       <entry>back-up, back up</entry>
       <entry>noun</entry>
      </row>
+     <row>
+      <entry>bare metal</entry>
+      <entry>bare-metal, baremetal</entry>
+      <entry>noun; a computer system without a base operating system 
+             or installed applications</entry>
```
Comment (dariavladykina) [inline xml/docu_styleguide.terminology.xml:156]:
```suggestion
      <entry>noun; a computer without an operating system; <emphasis>also</emphasis> a physical computer (in 
       contrast to a virtualized system)</entry>
```

**Review by dariavladykina — COMMENTED** [review]
Review changes applied

**Pull request merged**